regenerate-test-cpgs.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+
+export CPG_ROOT_DIR=`pwd`
+
+# https://stackoverflow.com/a/28085062
+: ${DEFAULT_JAVA2CPG_ROOT:=`readlink -f ../java2cpg`}
+# : ${DEFAULT_JAVA2CPG_ROOT:=${readlink -f "../java2cpg"}}
+echo "java2cpg root directory [$DEFAULT_JAVA2CPG_ROOT]:"
+read INSTALL_DIR
+if [ -z "$JAVA2CPG_ROOT" ]; then
```

Comment:
Please rename to JAVA2CPG_HOME to have this consistent with naming of ENV variable for testing in CS repo.

Reply:
done